### PR TITLE
List group permission for repo

### PR DESCRIPTION
**Pull request by j0nathan33**

### CODE REVIEW

**Comment by cdancy** [comment]
Thanks @j0nathan33 !

**Comment by j0nathan33** [comment]
Ok thanks, I will do change in my side :)

File: src/main/java/com/cdancy/bitbucket/rest/features/RepositoryApi.java [hunk 1]
```diff
@@ -82,4 +83,15 @@ boolean delete(@PathParam("project") String project,
...
                         @Nullable @QueryParam("start") Integer start,
                         @Nullable @QueryParam("limit") Integer limit);
+
+    @Named("repository:list-Permissions-Group")
+    @Documentation({"https://developer.atlassian.com/static/rest/bitbucket-server/5.0.0/bitbucket-rest.html#idm45659054969200"})
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/{project}/repos/{repo}/permissions/groups")
+    @Fallback(BitbucketFallbacks.PermissionsPageOnError.class)
+    @GET
+    PermissionsPage listPermissionsGroup(@PathParam("project") String project,
```
Comment (cdancy) [inline src/main/java/com/cdancy/bitbucket/rest/features/RepositoryApi.java]: @j0nathan33 FYI ... I'm going to rename this on master to `listGroupPermissions`.

Comment (cdancy) [inline src/main/java/com/cdancy/bitbucket/rest/features/RepositoryApi.java]: Ignore that ... looks like we already did something similar :)

**Pull request merged**